insights/parsers/irqbalance_conf.py

```diff
@@ -0,0 +1,46 @@
+"""
+irqbalance_conf - Contents of ``/etc/sysconfig/irqbalance``
+=============================================================
```

Comment:
Normally we use `File` instead of `Contents of` here.
Better keep the "=" in the same length with the title.